### PR TITLE
app-misc/c_rehash: version bump to 1.7_p20160204

### PR DESCRIPTION
**Pull request by Xiami2012**
Also update the EAPI to 6

Since we care openssl-c_rehash.sh only and upstream has migrated to Git
now, the versioning scheme may be improper. Use pXXXXXXXX (X means
commit date) now and follow the old scheme for compability (as we don't
have epoch field).

Package-Manager: Portage-2.3.51, Repoman-2.3.11
Signed-off-by: Xiami <i@f2light.com>

### CODE REVIEW

File: app-misc/c_rehash/c_rehash-1.7_p20160204.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_FILE="openssl-c_rehash.sh"
+GIT_COMMIT="f79908c1f7004d0abd384451abd923c6b6ba323d"
+
+DESCRIPTION="c_rehash script from OpenSSL"
+HOMEPAGE="https://www.openssl.org/ http://git.pld-linux.org/?p=packages/openssl.git https://github.com/pld-linux/openssl/"
+SRC_URI="http://git.pld-linux.org/?p=packages/openssl.git;a=blob_plain;f=${MY_FILE};hb=${GIT_COMMIT} -> ${MY_FILE}.${PV}"
```
Comment (Whissi) [inline app-misc/c_rehash/c_rehash-1.7_p20160204.ebuild]: PID-Linux.org is available via HTTPS. Please Update HOMEPAGE and SRC_URI.

Comment (Xiami2012) [reply app-misc/c_rehash/c_rehash-1.7_p20160204.ebuild]: Sorry for that. Updated.

**Comment by Xiami2012** [comment]
> 
> 
> But please notice that we won't be able to merge this before May 2019, see https://wiki.gentoo.org/wiki/Project:Base#Rules_and_limitations

Sorry, I didn't know the rule before. :cry: I downgraded it to EAPI 6 just now.

Now the diff is:
```diff
--- c_rehash-1.7-r1.ebuild      2019-01-16 17:05:24.730977653 +0800
+++ c_rehash-1.7_p20160204.ebuild       2019-01-25 19:54:10.447803709 +0800
@@ -1,15 +1,18 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2

-EAPI=5
+EAPI=6
+
+MY_FILE="openssl-c_rehash.sh"
+GIT_COMMIT="f79908c1f7004d0abd384451abd923c6b6ba323d"

 DESCRIPTION="c_rehash script from OpenSSL"
-HOMEPAGE="https://www.openssl.org/ https://github.com/pld-linux/openssl/"
-SRC_URI="https://cvs.pld-linux.org/cgi-bin/cvsweb.cgi/packages/openssl/openssl-c_rehash.sh?rev=${PV} -> openssl-c_rehash.sh.${PV}"
+HOMEPAGE="https://www.openssl.org/ https://git.pld-linux.org/?p=packages/openssl.git https://github.com/pld-linux/openssl/"
+SRC_URI="https://git.pld-linux.org/?p=packages/openssl.git;a=blob_plain;f=${MY_FILE};hb=${GIT_COMMIT} -> ${MY_FILE}.${PV}"

 LICENSE="openssl"
 SLOT="0"
-KEYWORDS="alpha amd64 arm arm64 hppa ia64 m68k ~mips ppc ppc64 s390 sh sparc x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris ~x86-winnt"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~ppc-aix ~x64-cygwin ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~m68k-mint ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris ~x86-winnt"
 IUSE=""

 RDEPEND="!<dev-libs/openssl-1.0.2d-r1:0"
@@ -22,8 +25,10 @@
        sed \
                -e "/^DIR=/s:=.*:=${EPREFIX}${SSL_CNF_DIR}:" \
                -e "s:SSL_CMD=/usr:SSL_CMD=${EPREFIX}/usr:" \
-               "${DISTDIR}"/openssl-c_rehash.sh.${PV} \
+               "${DISTDIR}"/${MY_FILE}.${PV} \
                > "${WORKDIR}"/c_rehash || die #416717
+
+       eapply_user
 }

 src_install() {
```

**Comment by ajakk** [comment]
If you're still interested in this please update your ebuild to EAPI=7 and such, and then ping someone in base system.

**Comment by scardracs** [comment]
@ajakk it's possible to close this PR https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=5e3bef0cc369d7eed26769ea6bc3419046220037

**Comment by Xiami2012** [comment]
Seems c_rehash in ::gentoo is newer than this PR already?

**Comment by scardracs** [comment]
> Seems c_rehash in ::gentoo is newer than this PR already?

Yes, probably because this PR is from 2019 😅

**Pull request closed without merging**